config.go

```diff
@@ -181,26 +186,19 @@ func init() {
 			Outputs:     map[string]string{"console": ""},
 		},
 	}
-	ParseConfig()
-}
 
-// ParseConfig parsed default config file.
-// now only support ini, next will support json.
-func ParseConfig() (err error) {
-	if AppConfigPath == "" {
-		// initialize default configurations
-		AppConfigPath = filepath.Join(AppPath, "conf", "app.conf")
-		if !utils.FileExists(AppConfigPath) {
-			AppConfig = &beegoAppConfig{config.NewFakeConfig()}
-			return
-		}
+	AppConfigPath = filepath.Join(AppPath, "conf", "app.conf")
```

Comment:
it doesn't make sense. As if this variable still as public. but beego will never use it. As we overwrite it here. And I think we need to just build failed. and let the developer call LoadAppConfig